src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -99,6 +98,11 @@ public void driveArcade(double speed, double turn) {
         robotDrive.arcadeDrive(-speed * DRIVE.SPEED_MULTIPLIER, -turn * DRIVE.TURN_MULTIPLIER);
     }
 
+    public void tankDriveVolts(double leftVolts, double rightVolts) {
```

Comment:
Why do we have this?

Reply:
Generally, we shouldn't be controlling directly using voltage.

Reply:
I added this mainly because the wpi documentation has this for autonomous .

Reply:
Ah okay, it might be useful.